certified-connectors/AIForged/apiProperties.json

```diff
@@ -0,0 +1,66 @@
+{
+  "publisher": "Larc AI (PTY) Ltd",
+  "stackOwner": "Larc AI (PTY) Ltd",
+  "contact": {
+    "name": "AIForged",
+    "url": "https://www.aiforged.com",
+    "email": "info@aiforged.com"
+  },
+  "iconBrandColor": "#0E76BC",
+  "capabilities": [
+    "actions"
+  ],
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "AIForged",
+          "clientId": "swaggerui",
```

Comment:
@aiforged Please remove the client id value as per Microsoft's security policy. You can replace it with dummy text.
`"clientId": "[[DUMMY]]"`